ironfish/src/wallet/account.ts

```diff
@@ -108,14 +108,8 @@ export class Account {
     transactionHash: Buffer
     spent: boolean
   }> {
```

Comment:
We could change the return type to `AsyncGenerator<DecryptedNoteValue>`

Reply:
changed